fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/activity/ActivityLogRestClient.kt

```diff
@@ -180,6 +186,34 @@ class ActivityLogRestClient(
         }
     }
 
+    suspend fun fetchActivityTypes(remoteSiteId: Long, after: Date?, before: Date?): FetchedActivityTypesResultPayload {
+        val url = WPCOMV2.sites.site(remoteSiteId).activity.count.group.url
+        val params = mutableMapOf<String, String>()
+        addDateRangeParams(params, after, before)
+
+        val response = wpComGsonRequestBuilder.syncGetRequest(
+                this,
+                url,
+                params,
+                ActivityTypesResponse::class.java
+        )
+        return when (response) {
+            is Success -> {
+                buildActivityTypesPayload(response.data, remoteSiteId)
+            }
```

Comment:
Nitpick: Short branches can be put on the same line as the condition, without braces.

```suggestion
            is Success -> buildActivityTypesPayload(response.data, remoteSiteId)        
```

Reply:
Fixed in 8d419ce2d4b05ba8c61c0cba6c6dc133d3bc0093